express-notes-api/deleteNote.js

```diff
@@ -0,0 +1,29 @@
+function deleteNote(app) {
+  const fs = require('fs');
+  const data = require('./data.json');
+  let dataJSON = null;
+  app.delete('/api/notes/:id', (req, res) => {
```

Comment:
It would be best to put the calls that set up your endpoints in one place, and let this function just do the work that deletes the note.